lldb/source/Breakpoint/BreakpointLocation.cpp

```diff
@@ -524,6 +524,9 @@ void BreakpointLocation::GetDescription(Stream *s,
           s->EOL();
           s->Indent("function = ");
           s->PutCString(sc.function->GetName().AsCString("<unknown>"));
+          s->EOL();
+          s->Indent("mangled function = ");
+          s->PutCString(sc.function->GetMangled().GetMangledName().AsCString("<unknown>"));
```

Comment:
```
if (sc.function->GetMangled().GetMangledName()) {
  s->Indent("mangled function = ");
  s->PutCString(sc.function->GetMangled().GetMangledName().AsCString("<unknown>"));
}
```

Reply:
Does SymbolContext::GetFunctionName do the right thing here?  It will also find the innermost inlined function if the SC is in some inlining and return that as well.

Reply:
If it does, you could just call it twice, once with the prefer mangled false, and once true.

Reply:
> It will also find the innermost inlined function if the SC is in some inlining and return that as well.

From an inspection of its implementation, it is not _as well_, but rather _instead_.

> It might be nice in verbose mode to see the whole chain of functions from the concrete function down to the most inlined?

I agree it would be nice, but I couldn't find any readily available APIs that go past the first inline context, so this is probably better done as a follow up

Reply:
We might be able to reuse `bool SymbolContext::DumpStopContext`, this is what is the regular `break list` uses

Reply:
It does look like `SymbolContext::DumpStopContext` does what we want. Will the output fit well into the "breakpoint list --verbose" output? Or will it look too different?

I would be fine to just print the inline function name if the output is too different. 

FYI: Any `lldb_private::Block` may or may not have inlined function info. You can always call:
```
Block *inlined_block = block->GetContainingInlinedBlock();
```
If `block` is an inlined function it will return `block`, else it will get the parent block and see if any parent blocks contain inline function info. If it returns NULL, then `block` isn't a or contained in an inline block. 

Getting an inline call stack from this information is tricky and is done in `SymbolContext::DumpStopContext()` correctly.

Reply:
Now that you mention it, I think we kind of _have_ to use `DumpStopContext`.  As it stands, the verbose output is missing information that the non-verbose output has, which feels wrong.

But let's merge this one first, since doing the above requires changing existing pieces, and this PR is about adding a new piece.